funcs.py

```diff
@@ -9,3 +9,6 @@ def mult(x, y):
 def bye():
     ''':('''
     exit()
+
+def subtract(x, y):
+    return x -y
```

Comment:
```suggestion
    return x - y
```
Watch the formatting